### PR TITLE
Release 0.14.2

### LINKED ISSUE #401: Page endpoints with many samples
This endpoint needs to be locked down; it's giving way too much stuff, which crashes the server for large projects: https://pephub-api.databio.org/api/v1/projects/databio/bedbase_raw/samples?tag=default&raw=false

anything listing a huge number of things has to be paged.

### PR DESCRIPTION
**Pull request by khoroshevskyi**
## Changes:
- Fixed #401

### CODE REVIEW

File: web/src/components/project/validation/validation-result.tsx [hunk 1]
```diff
@@ -2,23 +2,25 @@ import { useState } from 'react';
...
-  const { validationResult, isValidating, schemaRegistry } = props;
+  const { validationResult, isValidating, schemaRegistry, shouldValidate } = props;
 
   const [validationModalIsOpen, setValidationModalIsOpen] = useState(false);
 
   let wrapperClassName = 'py-1 px-2 rounded-1 bg-opacity-10 validation-button';
-  if (isValidating) {
+  if (!shouldValidate) {
+    wrapperClassName += ' border border-warning text-warning bg-warning';
+  } else if (isValidating) {
```
Comment (sanghoonio) [inline web/src/components/project/validation/validation-result.tsx:23]: shouldValidate is returning false only so the validation button never changes color from yellow.. Color change works if you set the if statement to check for shouldValidate and not !shouldValidate..

File: pephub/routers/api/v1/project.py [hunk 2]
```diff
@@ -93,7 +94,12 @@ async def get_namespace_projects_list(
     return agent.annotation.get_by_rp_list(registry_paths=paths, admin=namespace_access)
 
 
-@project.get("", summary="Fetch a PEP", response_model=ProjectRawRequest)
+@project.get(
+    "",
+    summary="Fetch a PEP",
+    response_model=ProjectRawRequest,
+    response_model_by_alias=False,
```
Comment (nleroy917) [inline pephub/routers/api/v1/project.py:105]: Just curious, so I can learn... whats this for?

**Review by nleroy917 — APPROVED** [review]
👍🏼

File: pephub/routers/api/v1/project.py (hunk 2 above)
Comment (khoroshevskyi) [reply pephub/routers/api/v1/project.py:105]: pydantic has aliases, and by default, fastapi in openAPI docs is using them. I don't want to use aliases, but actual names, that's why I set it to False

**Comment by khoroshevskyi** [comment]
@nleroy917 @sanghoonio Updated bedms, could you please test it and confirm if everything works correctly?

**Pull request merged**